### ISSUE TITLE
How to inspect attributes of newly generated features?

### ISSUE THREAD

**Issue by avillasmil** [IIIS-Li-Group/OpenFE#31]
I am looking to for a way to determine the bin edges that are computed for freq(x) features? Once the object is fit using the fit() method where is this information stored within the object?  Thanks!

**Comment by ZhangTP1996** [comment]
Numerical features are not binned to perform freq(x). The feature values are directly transformed by using pd.Series.value_counts()

**Comment by avillasmil** [comment]
As per the documentation of value_counts(), when continuous variables are
inputted, bins are leveraged in order to transform into categorical
variables.

So when the test set is being transformed using the freq() operator on
continuous non-integer data, are the bins that are being used established
via the distribution in the test set or the training set?

Thanks!

**Comment by ZhangTP1996** [comment]
> 

According to the documentation https://pandas.pydata.org/docs/reference/api/pandas.Series.value_counts.html, `bins` is optional and set to `None `in default?
